Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/BPMNEditorController.java

```diff
@@ -198,6 +198,7 @@ public BPMNEditorController() {
         param.put("bpmnioLib", AccessType.VIEWER.equals(currentUserAccessType) ? BPMNIO_VIEWER_JS : BPMNIO_MODELER_JS);
       }
       param.put("viewOnly", AccessType.VIEWER.equals(currentUserAccessType));
+      param.put("disableSimulateModel", mainC.getPortalPluginMap().get(PluginCatalog.PLUGIN_SIMULATE_MODEL) == null);
       Executions.getCurrent().pushArg(param);
```

Comment:
A better name for "disableSimulateModel" is "availableSimulateModelPlugin" which is related to the checked condition. When this param is passed and checked in the .zul file below, it would be if (${arg.availableSimulateModelPlugin})) which is easier to understand why the button needs to be disbled.